docs/deploying.md

```diff
@@ -64,59 +24,36 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
+    permissions:
+      pages: write
+      id-token: write
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: 22
           cache: npm
       - run: npm ci
       - run: npm run build
-      - name: Deploy to Observable Cloud
-        # This parameter to `--message` will use the latest commit message
-        run: npm run deploy -- --message "$(git log -1 --pretty=%s)"
-        env:
-          # Authentication information. See below for how to set this up.
-          OBSERVABLE_TOKEN: ${{ secrets.OBSERVABLE_TOKEN }}
+      - uses: actions/configure-pages@v4
+      - uses: actions/upload-pages-artifact@v3
+        with:
+          path: src/.observablehq/dist
```

Comment:
tried gh action and it failed:

<img width="300" alt="0E356283-4AC5-4115-B96B-581624970546-22166-000C47A298D3B0B3" src="https://github.com/user-attachments/assets/43c46ee6-f96f-41e3-863d-5388869de6c5" />

i think for most framework projects this'll just be /dist right? i think maybe it's weird and different in the framework repo itself

```suggestion
          path: dist
```

trying now…

Reply:
hm that just builds the markdown readme. i think we need the actual build command somewhere in the workflow file? https://www.tophtucker.com/tophtown/ 

<img width="300" alt="image" src="https://github.com/user-attachments/assets/3118726f-03ea-41fc-b5d3-1e0e7e17c924" />

Reply:
No, `src/.observablehq/dist` should be right. Are you sure you don’t have the source root set to `docs`? In which case, you’ll need to edit the action to use `docs/.observablehq/dist` instead.